src/dispatcher.js

```diff
@@ -60,36 +61,36 @@ class Dispatcher extends machina.BehavioralFsm {
 			actionMap: {},
 			states: {
 				ready: {
-					_onEnter: function() {
+					_onEnter: () => {
 						this.actionContext = undefined;
 					},
 					"action.dispatch": "dispatching"
 				},
 				dispatching: {
-					_onEnter: function( luxAction ) {
+					_onEnter: luxAction => {
```

Comment:
Per the above comment, you'll want to change the arrow functions to method shorthand on the rest of this object.